packages/edit-navigation/src/components/menu-delete/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+export default function MenuDelete( { menuId, onDelete } ) {
+	const deleteMenu = async ( recordId ) => {
+		const path = `${ '/__experimental/menus/' + recordId + '?force=true' }`;
```

Comment:
Using both `` for string interpolation and + for string concatenation is unnecessary.

Pick one:

```js
const path = '/__experimental/menus/' + recordId + '?force=true';
```

```js
const path = `/__experimental/menus/${ recordId }?force=true`;
```

Reply:
long live copypasta and tomatoes.